Makefile.PL

```diff
@@ -34,7 +34,11 @@ if ($^O eq 'MSWin32') {
 my $cc_option_flags = '-O2 -g -Wall -Werror';
 
 if ($Config{gccversion} =~ /llvm/i) {
-  $cc_option_flags .= ' -Wno-deprecated-declarations'; # -Wno-compound-token-split-by-macro
+  if ( $^O eq 'darwin' && $Config{gccversion} =~ /LLVM 12.0.5/) {
```

Comment:
HI @timlegge this file is auto-generated now by `main/Makefile_header.PL` right? so these changes will be overwritten, by the changes proposed below

Reply:
@jonasbn Makefile.PL is automatically created by dzil build and copied over the existing.  Either way since Makefile.PL is in the repo you will need the changes.

I did the build which created the updated Makefile.PL and then committed the changes